### DISCUSSION TITLE
How to immediately open the terminal

### DISCUSSION THREAD

**Discussion by dimas-cyriaco** [stevearc/overseer.nvim#226] [Q&A] [answered]
Can I configure Overseer to automatically open the terminal when I select the template?

**Comment by stevearc** [comment, marked as answer]
#68 has a solution if you're calling it via the API. If you want it to happen whenever you do `:OverseerRun` you could create a [custom component](https://github.com/stevearc/overseer.nvim/blob/master/doc/guides.md#custom-components) that does the same thing in `on_start`, and attach that component to all tasks by adding it to the `default` [component alias](https://github.com/stevearc/overseer.nvim/blob/master/doc/guides.md#component-aliases)

**Reply by dimas-cyriaco** [reply]
The tip on https://github.com/stevearc/overseer.nvim/issues/68 worked perfectly, thank you.
